grabbit/extensions/writable.py

```diff
@@ -17,22 +17,40 @@ def replace_entities(entities, pattern):
     Args:
         entities (dict): A dictionary mapping entity names to entity values.
         pattern (str): A path pattern that contains entity names denoted
-            by square brackets.
+            by curly braces. Optional portions denoted by square braces.
+            Accepted entity values denoted within angle brackets.
```

Comment:
Trivial edit, but can you mention that this uses regex matching, so any valid regex can be passed inside the angle brackets?